core/test/unit/models/post_spec.js

```diff
@@ -1890,4 +1890,84 @@ describe('Unit: models/post', function () {
             });
         });
     });
+
+    describe('Mobiledoc conversion', function () {
+        let config = require('../../../server/config');
+        let origConfig = _.cloneDeep(config);
+        let labs = require('../../../server/services/labs');
+        let origLabs = _.cloneDeep(labs);
+        let sandbox = sinon.sandbox.create();
```

Comment:
This is redundant. The sandbox is restored `afterEach` for each test run, see top.